### ISSUE TITLE
Move the .cht definitions out of colorchecker.h to textfiles

### ISSUE THREAD

**Issue by TutajITeraz** [darktable-org/darktable#17855]
**Is your feature request related to a problem? Please describe.**
I cannot add my own pigment-based checker card to darktable.

**Describe the solution you'd like**
Half solution: 
Move the .cht definitions out of colorchecker.h to textfiles (as a csv, json or .cht file) so everyone could add own color calibration cards by adding new files to the folder.

Full solution:
Add GUI color checker card editor. It could be usefull for the darktable-chart too.

PLUS:
Add button to run darktable-chart from main darktable app. (Usefull for flatpack version).

**Alternatives**
Now i need to edit the src/common/colorchecker.h file manually and recompile darktable. It is not possible due to #17854 issue.

**Comment by Donatzsky** [comment]
Already discussed - and rejected - in #8705

**Comment by Jiyone** [comment]
It could be the same as the in/out color profiles, those added in the folder will be loaded in the module's profile list. 
I use transparency IT8 charts for slide film and that could help.

**Comment by jenshannoschwalm** [comment]
> It could be the same as the in/out color profiles, those added in the folder will be loaded in the module's profile list.
> I use transparency IT8 charts for slide film and that could help.

Agreed. If someone does a working PR i would certainly support that.

**Comment by Jiyone** [comment]
I already tried to code something some years ago but my skills were not good enough 😅

**Comment by jenshannoschwalm** [comment]
Maybe they have improved ?

**Comment by jenshannoschwalm** [comment]
Maybe even some dev on some dt fork might be interested ?

**Comment by victoryforce** [comment]
> They have but this will still take time to code.

Everyone will have to spend time coding. But others don't even have a personal interest.

> I don't know who might be interested, this is a feature for film photographer using an IT8 chart and they are a few.

The code for implementing the functionality requested in this issue is fundamentally simple... the only problem is niche nature of this functionality. Therefore, active contributors will most likely not take on it, because they always have more priority tasks.

This is exactly the situation when the "scratch your own itch" principle applies.
